app.coffee

```diff
@@ -38,6 +38,7 @@ app.param 'doc_id', (req, res, next, doc_id) ->
 		next new Error("invalid doc id")
 
 app.get    '/project/:project_id/doc/:doc_id',                          HttpController.getDoc
+app.get    '/project/:project_id',                                      HttpController.getProjectDocs
```

Comment:
`/project/:project_id/doc` would be more consistent with our routes for a 'list docs' end point.